src/main/java/pl/agawesolowska/ticketbookingapp/api/BookingRestController.java

```diff
@@ -0,0 +1,32 @@
+package pl.agawesolowska.ticketbookingapp.api;
+
+import javax.validation.Valid;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import pl.agawesolowska.ticketbookingapp.facade.BookingFacade;
+import pl.agawesolowska.ticketbookingapp.model.dto.BookingRequestDTO;
+import pl.agawesolowska.ticketbookingapp.model.dto.BookingResultDTO;
+
+/**
```

Comment:
po co ten komentarz? czy nie ma tego w Gicie?

Reply:
Jasne, jak najbardziej w Gicie jest informacja co, kto i kiedy. To bardziej moje przyzwyczajenie z podpisywaniem kodów wrzucanych na GitHuba, zazwyczaj dodaję jeszcze czym jest ta klasa/interfejs/do czego służy/itp. Tutaj to pominęłam, bo znacie założenia projektu.